_apps/prr-review-counter.md

```diff
@@ -0,0 +1,19 @@
+---
+# A human-friendly name of your listing
+title: Pull Request Review Counter
+# A short description of what your app does
+description: Require multiple approved reviews before merging a Pull Request.
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: pull-request-review-counter
+# Include a few screenshots that show your app in action
+screenshots:
+- https://user-images.githubusercontent.com/82819/37225532-c290047e-238b-11e8-9443-0322d16cc0b1.png
```

Comment:
Could you zoom this out a bit so it includes the whole checks box? Similar to https://probot.github.io/apps/dco/

It'd also be great to have a second screenshot of it failing.

Reply:
![phani1](https://user-images.githubusercontent.com/10412121/37304553-e59a0f34-25ff-11e8-81f8-e7ace8f92d75.png)
:)